src/state/slices/assetsSlice/assetsSlice.ts

```diff
@@ -77,14 +77,14 @@ export const assetApi = createApi({
         data && dispatch(assets.actions.setAssets(data))
       },
     }),
-    getAssetDescription: build.query<AssetsState, AssetId>({
-      queryFn: async (assetId, { getState }) => {
+    getAssetDescription: build.query<AssetsState, { assetId: AssetId; selectedLocale: string }>({
+      queryFn: async ({ assetId, selectedLocale }, { getState }) => {
         const service = await getAssetService()
         // limitation of redux tookit https://redux-toolkit.js.org/rtk-query/api/createApi#queryfn
-        const { byId: byIdOriginal, ids } = (getState() as any).assets as AssetsState
+        const { byId: byIdOriginal, ids } = (getState() as ReduxState).assets as AssetsState
```

Comment:
My OCD wrote this, will remove it from this PR to keep things clean. Looks like we need a `getAssets` selector to remove this casting.
Pinging myself @gomesalexandre to tackle it right after